server/src/services/logging/RequestData.ts

```diff
@@ -0,0 +1,138 @@
+export type ConnectionMetaData = {
```

Comment:
you don't need most of the types in this file, you can delete them and update the rest of the changes according to it